### ISSUE TITLE
Feature Request: Throw hand away instead of playing a card

### ISSUE THREAD

**Issue by raidekoptix** [ajanata/PretendYoureXyzzy#189]
In the rules of CAH you can throw your entire hand away and redraw instead of playing a card. Can we implement this in PYX?

**Comment by Benji-Collins** [comment]
That would be cool. After a few rounds you kind of accumulate a number of bad cards.

**Comment by devgianlu** [comment]
> **Rebooting the universe:** At any time, players may trade in an Awesome Point to return as many White Cards as they'd like to the deck and draw back up to ten.

I may start working on this when I have time.

**Comment by raidekoptix** [comment]
Yes! Thank you! I'm totally willing to help, but I'm not much of a dev, and haven't had the time to try and deep-dive into the project and figure out what's what... I can't promise anything, but if you're familiar with the codebase already and can offer me some microwave-instructions, I might be able to assist?

**Comment by devgianlu** [comment]
I think I'm not going for this since the owner won't ever accept a PR that big.

**Comment by ajanata** [comment]
It's definitely one of the things I'd like to have done at some point, but I don't think I'd ever have time to do it entirely myself.
